contracts/modules/spigot/Spigot.sol

```diff
@@ -1,382 +0,0 @@
-pragma solidity 0.8.9;
-
-import { ReentrancyGuard } from "@openzeppelin/contracts/security/ReentrancyGuard.sol";
-import { LineLib } from  "../../utils/LineLib.sol";
-
-import {ISpigot} from "../../interfaces/ISpigot.sol";
-
-/**
- * @title Spigot
- * @author Kiba Gateaux
- * @notice Contract allowing Owner to secure revenue streams from a DAO and split payments between them
- * @dev Should be deployed once per line. Can attach multiple revenue contracts
- */
-contract Spigot is ISpigot, ReentrancyGuard {
```

Comment:
Why delete the Spigot and Escrow files? We still need to create external functions in their contracts that hook into library functions.